src/app/features/shows/shows.service.ts

```diff
@@ -0,0 +1,34 @@
+import { environment } from './../../../environments/environment.development';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import queryString from 'query-string';
+import { map, Observable } from 'rxjs';
+import camelcaseKeys from 'camelcase-keys';
+import { SearchRequest } from './type/search-request.type';
+import { ShowList } from './models/show-list.model';
+import { Show } from './models/show.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ShowsService {
+  constructor(private http: HttpClient) {}
+
+  getTrending(
+    request: SearchRequest,
+    type: string
```

Comment:
```suggestion
    mediaType: string
```
type là từ khóa rồi e